### PR TITLE
Public acct page

### PR DESCRIPTION
**Pull request by alexmiddeleer**

### CODE REVIEW

File: postgres/migrations/next-migration.sql [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+alter table mythgard.account add column profile_icon_id integer default 1;
 
 CREATE POLICY update_account_if_moderator
   ON mythgard.account
   FOR UPDATE
   USING (exists(select * from mythgard.account_moderator
          where account_id = mythgard.current_user_id()));
 
-GRANT UPDATE (account_type) ON TABLE mythgard.account TO authd_user;
+GRANT UPDATE (account_type, profile_icon_id) ON TABLE mythgard.account TO authd_user;
```
Comment (alexmiddeleer) [inline postgres/migrations/next-migration.sql:12]: not yet used.

**Comment by alexmiddeleer** [comment]
I should write a cypress test for this I think, at least to check that the page loads.

File: next/components/profile.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      <style jsx>{`
+        .user-profile {
+          text-align: center;
+        }
+        .profile-image {
+          width: 138px;
+          margin-top: 30px;
+        }
+        .user-name {
+          color: #f1810b;
```
Comment (lily-pytel) [inline next/components/profile.js:25]: Any reason not to use our theme?

Comment (alexmiddeleer) [reply next/components/profile.js:25]: Uh, none - I just copy pasted this from the old page.

Comment (alexmiddeleer) [reply next/components/profile.js]: Fixed, good call.

**Pull request merged**